src/docs/reference/modules/Microsoft.Authentication/README.md

```diff
@@ -23,27 +23,27 @@ If no value is provided, setup Microsoft Account app to use the default path /si
 
 ## Microsoft Entra ID (Azure Active Directory)
 
-Authenticates users with their Microsoft Entra ID Account, like Microsoft work or school accounts. If the site allows to register new users, a local user is created and the Microsoft Entra ID account is linked. If a local user with the same email is found, then the external login is linked to that account, after authenticating.
+Authenticates users with their Microsoft Entra ID Account, including Microsoft work, school, and personal accounts (such as Skype, Xbox, Outlook.com). If the site allows to register new users, a local user is created and the Microsoft Entra ID account is linked. If a local user with the same email is found, then the external login is linked to that account, after authenticating.
```

Comment:
How does this feature now contrast with the Microsoft Account one? Document it under both sections.

Reply:
Isn't that clear enough? One is for Microsoft accounts only, the other one for Entra ID with the optional support for personal accounts that it offers.

Reply:
I don't think it is. The original documentation is not necessarily completely clear, to begin with (since an Entra ID account is also a "Microsoft account"), but now there's also a feature overlap too.

Reply:
I added some notes, not sure if it sufficient.

Reply:
Will be OK.